components/server/ee/src/gitlab/gitlab-app-support.ts

```diff
@@ -61,4 +75,27 @@ export class GitLabAppSupport {
         result.unshift(...ownersRepos);
         return result;
     }
+
+    protected async getAccountAvatarUrl(project: ProjectSchema, providerHost: string): Promise<string> {
+        const owner = project.owner || project.namespace;
+        if (owner.avatar_url) {
+            const url = owner.avatar_url;
+            // Sometimes GitLab avatar URLs are relative -- ensure we always use the correct host
+            return url[0] === "/" ? `https://${providerHost}${url}` : url;
+        }
+        // If there is no avatar, generate the same default avatar that GitLab uses. Based on:
+        // - https://gitlab.com/gitlab-org/gitlab/-/blob/b2a22b6e85200ce55ab09b5c765043441b086c96/app/helpers/avatars_helper.rb#L151-161
+        // - https://gitlab.com/gitlab-org/gitlab-foss/-/blob/84b4743475246e91dc78c3f25f9b335c40be84cd/app/assets/stylesheets/startup/startup-general.scss#L1611-1631
+        // - https://gitlab.com/gitlab-org/gitlab-foss/-/blob/84b4743475246e91dc78c3f25f9b335c40be84cd/app/assets/stylesheets/startup/startup-general.scss#L420-422
```

Comment:
Good point. I guess we could somehow re-use the same fonts as in our dashboard, but:
- Since they're not exactly the same as GitLab, I'm worries that in some cases they might look different than GitLab icons (which fails to achieve our goal here)
- Also, I'm not entirely sure how to reference the Tailwind class from inside the SVG (which doesn't use Tailwind) 🤔

So, maybe copying the fonts from GitLab just for the GitLab default avatar makes sense? 😇

Reply:
To clarify, the question aimed to suggest not importing any fonts but relying on our font stack to render the character inside the group avatar. We do use the same background color which already helps, right?

Reply:
This is what the SVG looks like for me when I remove the font specification:

<img width="158" alt="Screenshot 2022-03-30 at 15 49 22" src="https://user-images.githubusercontent.com/599268/160850253-edcc733c-7d50-4944-a5db-2a4595a1d93b.png">

Reply:
Oh, I meant rely on (use) our font stack instead. 😇